### PR TITLE
support single_translate in epub

### PR DESCRIPTION
**Pull request by hleft**
related
https://github.com/yihong0618/bilingual_book_maker/issues/231
https://github.com/yihong0618/bilingual_book_maker/issues/214
https://github.com/yihong0618/bilingual_book_maker/issues/170
https://github.com/yihong0618/bilingual_book_maker/issues/169

support `--single_translate` in epub

### CODE REVIEW

File: book_maker/cli.py [hunk 1]
```diff
@@ -230,6 +230,11 @@ def main():
         python3 "make_book.py" --book_name "test_books/animal_farm.epub" --retranslate 'test_books/animal_farm_bilingual.epub' 'index_split_002.html' 'in spite of the present book shortage which'
 """,
     )
+    parser.add_argument(
+        "--no_bilingual",
+        action="store_true",
+        help="output translated book, no bilingual",
```
Comment (yihong0618) [inline book_maker/cli.py:236]: --single_translate ? maybe better?

File: book_maker/cli.py [hunk 2]
```diff
@@ -308,6 +313,7 @@ def main():
         is_test=options.test,
         test_num=options.test_num,
         prompt_config=parse_prompt_arg(options.prompt_arg),
+        no_bilingual=options.no_bilingual,
```
Comment (yihong0618) [inline book_maker/cli.py:316]: if we add here we also need to add it in txt loader

Comment (hleft) [reply book_maker/cli.py:316]: I want people who use this loader to modify the loader they use.(I only use epub now) How about writing the currently supported loader in this help?

File: book_maker/cli.py (hunk 1 above)
Comment (hleft) [reply book_maker/cli.py:236]: ok I am going to modify after dinner

**Comment by yihong0618** [comment]
@hleft thanks

**Comment by phazei** [comment]
would be nice if this was added to the docs, it's not in the main readme

**Pull request merged**